src/commcare_cloud/ansible/deploy_cloudwatch_logs.yml

```diff
@@ -0,0 +1,118 @@
+
+- name: formplayer
+  hosts: formplayer
+  become: true
+  roles:
+    - role: cloudwatch_logs
+      vars:
+        log_files_collect_list:
```

Comment:
<img width="828" alt="Screen Shot 2020-08-26 at 3 42 18 PM" src="https://user-images.githubusercontent.com/615126/91350107-3a82c680-e7b4-11ea-9e33-dad1331e1665.png">

Reply:
@dannyroberts These are ready for your review.